beamer/config/commands.py

```diff
@@ -67,12 +67,7 @@ def _replay_event(w3: Web3, deployment: Deployment, config: Configuration, event
                 config.request_manager.tokens[symbol] = TokenConfig(
                     transfer_limit=event.transfer_limit, eth_in_token=event.eth_in_token
                 )
-
-                address = config.token_addresses.get(symbol)
-                if address is None:
-                    config.token_addresses[symbol] = event.token_address
-                else:
-                    assert address == event.token_address
+                config.token_addresses[symbol] = event.token_address
```

Comment:
I think we should not remove the `assert`. Please consider what happens if you have a token with `(FOO, 0x123)` (`(symbol, address)`), and then add another token, e.g. `(FOO, 0x456)`. Without the check, we would simply overwrite the existing entry, which is bad and something the check would have prevented. If such a cases happens, we would like to know about it.

Alternatively, we could handle such cases to use the address, rather than the symbol, in case a token with the same symbol already exists.

Reply:
Since we have no real way of actually manipulating the mapping, I think this needs to be discussed first.